modules/ROOT/Edits/create-inbound-message-flow.adoc

```diff
@@ -0,0 +1,119 @@
+* If your organization will receive the AS2 messages for the partner via a 3rd party connection, such as a VAN or 3PL, select the name of the 3rd party connection. The AS2 identifier and certificate of the selected 3rd party connection will be used in AS2 message validation.
+* If you selected AS2 as the receive to partner endpoint protocol, you must import the AS2 certificate of the partner or the 3rd party connection sending the AS2 message, if a certificate already does not exist in the partner or 3rd party connection’s profile.
+
+[source-message-type]
+== Configure the Source Message Type
+
+. In the message flow, expand the *Source message* section and click *Select*.
+. In the *Select message type* window:
+.. To use an existing message type, select the message type from the list and then click *Select*.
+.. To create a new message type, click *New* next to the search box, configure the X12 or EDIFACT xref:partner-manager-create-message-type.adoc[message type], and click *Save*.
+The message type and its validation settings are added to the message flow.
+. If your partner expects your organization to return functional acknowledgements:
+.. In the *Acknowledgement* section, select the check box next to *Send 997 functional acknowledgment (X12)* or *Send CONTRL functional acknowledgement (EDIFACT)* .
+.. Click *Select*.
+.. In the *Select an endpoint: Send acknowledgment to <partner>* window, click *Select* to specify the endpoint from which to receive the acknowledgment transactions from your partner.
++
+.. Alternatively, you can create a new send to partner endpoint:
+... Click *New*.
+... In the *Owner* field, select the endpoint owner.
+... In the *Protocol* field, select and configure the protocol.
+... Click *Save*.
++
+The acknowledgment is added to the message flow.
+
+[[import-map]]
+== Import the Translation Map
+
+To create your inbound EDI-to-application message (JSON or XML) map in Anypoint Studio, follow xref:partner-manager-maps.adoc[these instructions].
+
+. In the message flow, expand the *Map* section.
+. Click *Import* and then click *Choose file*.
+. In the *New mapping file* window, navigate to the DataWeave map you want to use and then click *Open*.
+. Click *Upload mapping*. +
+The translation map is added to the message flow.
+
+[[configure-target]]
+== Configure the Target
+
+. In the message flow, expand the *Target at <host>* section.
+. In the *Message Type* section, click *Select*.
+. In the *Select message type* window:
+* To use an existing message type, select the message type from the list and then click *Select*.
+* To create a new message type, click *New* next to the search box, define the xref:partner-manager-create-message-type.adoc#target-at-host[target at host message type], and click *Save*.
++
+The message type is selected for your backend system to use to map the EDI transaction. Partner Manager routes inbound EDI transactions based on matching identifiers configured at the Partner level.
```

Comment:
Is it really necessary to confirm that the steps had the desired effect? 
This might be concept info sneaking into task steps.